### PR TITLE
Search page sort

### LINKED ISSUE #8734: New sort component
See sort dropdown [in mockup](https://www.sketch.com/s/1cc6a024-668b-4d76-9617-115910d3a797/a/uuid/16B4B8A6-24CB-4F29-8F4B-CB8F889259B8), aligned to the left-hand side of pagination.

This one should be straightforward, with looks similar to what we go for for filters, without the checkboxes. The selected option, if there is one, should be the visible one.

On mobile, behaves like a normal dropdown (unlike filters who might open the side-modal)

### PR DESCRIPTION
**Pull request by AMBacelar**
## Who is this for?
Users of the new Search subpages

## What is it doing for them?
accessible sort component, and a support for no javascript

closes #8734

### CODE REVIEW

File: catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx [hunk 1]
```diff
@@ -96,9 +99,31 @@ const SearchLayout: FunctionComponent = ({ children }) => {
...
   return (
     <CataloguePageLayout {...pageLayoutMetadata}>
       <div className="container">
+        <form
+          role="search"
+          id="searchPageForm"
+          onSubmit={event => {
+            event.preventDefault();
+
+            trackEvent({
```
Comment (rcantin-w) [inline catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx:117]: Could we list all the added tracking events somewhere (maybe start a ticket) so we can discuss them with Tacey in our tracking catchups? It's good to add them, just want to double check how we want to structure the info within!

File: catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx [hunk 2]
```diff
@@ -96,9 +99,31 @@ const SearchLayout: FunctionComponent = ({ children }) => {
     }
   }, [currentSearchCategory]);
 
+  const updateUrl = (form: HTMLFormElement) => {
+    const urlQuery = formDataAsUrlQuery(form);
+    console.log(urlQuery);
```
Comment (rcantin-w) [inline catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx:104]: Leftover log?

File: common/views/components/SearchBar/SearchBar.tsx [hunk 3]
```diff
@@ -29,51 +28,34 @@ const SearchButtonWrapper = styled.div`
...
-            size="large"
-            colors={themeValues.buttonColors.yellowYellowBlack}
-          />
-        </SearchButtonWrapper>
-      </Container>
-    </form>
+    <Container>
+      <SearchInputWrapper>
+        <TextInputV2
+          id="dummy-searchbar"
```
Comment (rcantin-w) [inline common/views/components/SearchBar/SearchBar.tsx:38]: I know that's on me, but we should maybe change that id 😄

File: catalogue/webapp/pages/search/works.tsx [hunk 4]
```diff
@@ -42,44 +44,116 @@ export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
...
+        <div className="container">
           <Space
             v={{
               size: 'l',
               properties: ['padding-top', 'padding-bottom'],
             }}
           >
-            <ResultsPaginationWrapper>
-              {works.totalResults > 0 && (
-                <div>{works.totalResults} results</div>
```
Comment (rcantin-w) [inline catalogue/webapp/pages/search/works.tsx:76]: Did we remove this completely/Can we keep a placeholder?

**Comment by rcantin-w** [comment]
I'm very excited to see this in action! 🎉  Would it be worth making it its own component so the logic can be reused across all search pages?

File: catalogue/webapp/pages/search/works.tsx [hunk 5]
```diff
@@ -42,44 +44,126 @@ export const CatalogueSearchPage: NextPageWithLayout<Props> = ({
...
+  useEffect(() => {
+    if (router.query.sortOrder !== sortOrder) {
+      const form = document.getElementById('searchPageForm');
+      form?.dispatchEvent(
+        new window.Event('submit', { cancelable: true, bubbles: true })
+      );
+    }
+  }, [sortOrder]);
 
   if (works.totalResults === 0) return <p>nothing</p>;
```
[the remark this answers is not in the document]
Comment (AMBacelar) [reply catalogue/webapp/pages/search/works.tsx:68]: not currently I believe

[the remark this answers is not in the document]
Comment (rcantin-w) [reply catalogue/webapp/pages/search/works.tsx:68]: Hmmm you're right, I confused it with Images 🤔 It's something we'll have to align across at the end anyway, so not a blocker for here (but if you could leave a TODO comment that would be useful for future us!)

[the remark this answers is not in the document]
Comment (rcantin-w) [reply catalogue/webapp/pages/search/works.tsx:68]: Remembered this ticket I'd created that would probably relate: https://github.com/wellcomecollection/wellcomecollection.org/issues/8804

**Review by rcantin-w — APPROVED** [review]
🥎

**Pull request merged**